server/jvm/{{appName}}-site-specific/build.gradle.kts

```diff
@@ -69,42 +68,6 @@ publishing {
         }
     }
 }
-artifactory {
```

Comment:
Why have we removed this?

Reply:
same as what I mentioned above, I used simple app seed as a guide and thought there was no need for it, could return it though

Reply:
Fair enough. We might be able to remove it actually, we don't have any artifactory configuration in the server/jvm build.gradle.kts, which would be needed if you wanted to use artifactory.